lib/browserify.js

```diff
@@ -6,12 +6,7 @@ const transform = require('./js-transform')
 
 module.exports = browserifyIt
 
-function browserifyIt ({
-  config,
-  entry,
-  env,
-  minify
-}) {
+function browserifyIt({config, entry, env, minify}) {
```

Comment:
I'm not going to block it over this, yeah. Although I do question whether we want something that completely rewrites all our code.